tasks/publish-pyxis-repository/publish-pyxis-repository.yaml

```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: publish-pyxis-repository
   labels:
-    app.kubernetes.io/version: "0.2.0"
+    app.kubernetes.io/version: "0.3.0"
```

Comment:
@scoheb Sorry to bother you with this but as commit title says fix(), the version should be "0.2.1"

Reply:
fixed!